### PR TITLE
Bugfix/autostart broken android sdk

### PR DESCRIPTION
**Pull request by karimJWP**
checks for autostart in android sdk and made autoStartOnMobile more readable:

Note to reviewer: review after [PR in Android SDK](https://github.com/jwplayer/jwplayer-android-sdk-bishop/pull/691) is accepted
enables autostart in android sdk

Fixes #
JW7-3825

### CODE REVIEW

**Comment by karimJWP** [comment]
retest this please

**Comment by johnBartos** [comment]
test this please

**Comment by karimJWP** [comment]
@dannyfinks unit tests throw Unspecified error on IE 11 for intersection-observer.js:622 . is this being caused by my PR?

File: src/js/controller/model.js [hunk 1]
```diff
@@ -451,9 +451,13 @@ define([
...
 
         this.autoStartOnMobile = function() {
-            return this.get('autostart') && !this.get('sdkplatform') &&
-                ((_autoStartSupportedIOS() && (utils.isSafari() || utils.isChrome() || utils.isFacebook())) || (utils.isAndroid() && utils.isChrome())) &&
-                (!this.get('advertising') || this.get('advertising').autoplayadsmuted);
+            var autostartAdsIsEnabled = (!this.get('advertising') || this.get('advertising').autoplayadsmuted);
+            var iosBrowserIsSupported = _autoStartSupportedIOS() && (utils.isSafari() || utils.isChrome() || utils.isFacebook());
+            var androidBrowserIsSupported = utils.isAndroid() && utils.isChrome();
+            var mobileBrowserIsSupported = (iosBrowserIsSupported || androidBrowserIsSupported);
+            var isAndroidSdk = this.get('sdkplatform') === 1;
```
Comment (johnBartos) [inline src/js/controller/model.js]: This can be written as `this.get('sdkplatform')`

File: src/js/controller/model.js [hunk 2]
```diff
@@ -451,9 +451,13 @@ define([
...
+            var platformCanAutostart = (!this.get('sdkplatform') && autostartAdsIsEnabled && mobileBrowserIsSupported) || isAndroidSdk;
```
Comment (johnBartos) [inline src/js/controller/model.js]: What is `platform` in this context?

Comment (karimJWP) [inline src/js/controller/model.js]: platform meaning product: player, andorid sdk, ios sdk, etc..

**Pull request merged**